cmd/osv-scanner/main_test.go

```diff
@@ -349,59 +343,95 @@ func TestRun(t *testing.T) {
-									},
-									"locations": [
-										{
-											"physicalLocation": {
-												"artifactLocation": {
-													"uri": "fixtures/locks-many/package-lock.json"
-												}
-											}
-										}
-									]
-								}
-							]
-						}
-					]
-				}
+            {
+              "version": "2.1.0",
+              "$schema": "https://json.schemastore.org/sarif-2.1.0.json",
+              "runs": [
+                {
+                  "tool": {
+                    "driver": {
+                      "informationUri": "https://github.com/google/osv-scanner",
+                      "name": "osv-scanner",
+                      "rules": [
+                        {
+                          "id": "GHSA-whgm-jr23-g3j9",
+                          "shortDescription": {
+                            "text": "Uncontrolled Resource Consumption in ansi-html"
+                          },
+                          "fullDescription": {
+                            "text": "This affects all versions of package ansi-html. If an attacker provides a malicious string, it will get stuck processing the input for an extremely long time.",
+                            "markdown": "This affects all versions of package ansi-html. If an attacker provides a malicious string, it will get stuck processing the input for an extremely long time."
+                          },
+                          "help": {
+                            "text": "\n**Your dependency is vulnerable to [GHSA-whgm-jr23-g3j9](https://osv.dev/vulnerability/GHSA-whgm-jr23-g3j9)** \n.\n\n\n\u003e ## [GHSA-whgm-jr23-g3j9](https://osv.dev/vulnerability/GHSA-whgm-jr23-g3j9)\n\u003e \n\u003e This affects all versions of package ansi-html. If an attacker provides a malicious string, it will get stuck processing the input for an extremely long time.\n\u003e \n\n\n---\n\n### Affected Packages\n| Source | Package Name | Package Version |\n| --- | --- | --- |\n| lockfile:%%/fixtures/locks-many/package-lock.json | ansi-html | 0.0.1 |\n\n",
+                            "markdown": "\n**Your dependency is vulnerable to [GHSA-whgm-jr23-g3j9](https://osv.dev/vulnerability/GHSA-whgm-jr23-g3j9)** \n.\n\n\n\u003e ## [GHSA-whgm-jr23-g3j9](https://osv.dev/vulnerability/GHSA-whgm-jr23-g3j9)\n\u003e \n\u003e This affects all versions of package ansi-html. If an attacker provides a malicious string, it will get stuck processing the input for an extremely long time.\n\u003e \n\n\n---\n\n### Affected Packages\n| Source | Package Name | Package Version |\n| --- | --- | --- |\n| lockfile:%%/fixtures/locks-many/package-lock.json | ansi-html | 0.0.1 |\n\n"
+                          },
+                          "properties": {
+                            "deprecatedIds": [
+                              "GHSA-whgm-jr23-g3j9"
```

Comment:
This is the same as the actual ID? Should this be the case?

Reply:
Yes, I'm thinking deprecated Ids will contain all the IDs separately to avoid people using it to have separate logic to append displayId to it. This indicates that all the IDs are the same.

Reply:
Ack.